airlock/templates/file_browser/index.html

```diff
@@ -141,6 +141,17 @@
         {% endif %}
       {% endif %}
     {% /airlock_header %}
+
+    {% if release_request.is_under_review and user_has_reviewed_all_files and not user_has_completed_review %}
+      {% #airlock_todo_alert message="You have reviewed all files. You can now complete your review." %}{% /airlock_todo_alert %}
+    {% elif release_request.status.name == "REVIEWED" and is_output_checker and not is_author %}
+      {% if release_request.can_be_released %}
+        {% #airlock_todo_alert message="Two independent reviews have been completed. You can now return, reject or release this request." %}{% /airlock_todo_alert %}
+      {% else %}
+        {% #airlock_todo_alert message="Two independent reviews have been completed. You can now return or reject this request." %}{% /airlock_todo_alert %}
+      {% endif %}
+    {% endif %}
```

Comment:
I wonder if it would be cleaner overall to pull this logic out into the view and have it just pass the message into the template?

You might not even need a separate component then, which would reduce the level of indirection. Just something like:
```html
{% if todo_message %}
  <div class="mt-4">
    {% #alert variant="warning" title="Action required" dismissible=True %}
      {{ todo_message }}
    {% /alert %}
  </div>
{% endif %}
```